distribution/docs/src/main/resources/ManContents.adoc

```diff
@@ -221,6 +222,27 @@ Complete the following procedure now that several queries have been executed.
 Based on the browser's configuration, the .xls file will be downloaded or automatically displayed in Excel.
 ====
 
+=== Configuration Export/Import
+==== Export Configuration Settings
+
+1) Launch DDF.
+
+2) At the Admin Console, select any Active Application that you want to save.
+
+3) For each Active Application selected, click Save changes on the configuration that you want to save. This creates the configuration files.
+
+4) Using the DDF command line interface (CLI), type in config-export. The exported files will be exported to <DISTRIBUTION_INSTALL_DIR>/etc/exported
+
+5) Zip up the exported files in <DISTRIBUTION_INSTALL_DIR>/etc/exported
```

Comment:
We should add a callout section like this:

```
[IMPORTANT]
====
<Insert notes here>
====
```

That highlights the some of the important things in this:

> To keep the export/import process simple and consistent, and to make it possible for administrators to easily copy everything under the etc/exported directory, we will require all non-standard configuration files to be under the $DDF_HOME directory.
This means that all configuration files save from the Web Admin Console will be exported under etc/exported/etc, and the other non-standard files will be put under etc/exported followed by the relative path from $DDF_HOME.
For instance, if a keystore file is located under $DDF_HOME/keystores/keystore.jks, it will be exported to etc/exported/keystore/keystore.jks. This way, all an administrator will have to do is zip up or recursively copy all the files under etc/exported to the new DIB or DDF install home directory.